packages/contracts-bedrock/contracts/dispute/FaultDisputeGame.sol

```diff
@@ -34,7 +34,6 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, Semver {
     IBigStepper public immutable VM;
 
     /// @notice The duration of the game.
-    /// @dev TODO: Account for resolution buffer. (?)
```

Comment:
We're comfortable ignoring this for now?

Reply:
We've decided to add no resolution buffer for the game. There's a direct incentive to resolve the game in order to make bonds available, so there's not a concern about a game being left unresolved for long at all if we maintain a liveness assumption on the honest challenger 😄